src/CmSign.php

```diff
@@ -501,7 +500,7 @@ public function setBranding(string $kid, Branding $branding): Branding
     public function parseDocumentFieldsFile(string $path, string $documentId)
     {
         $fields = [];
-
+        
         $rows = SimpleExcelReader::create($path)->getRows()->all();
         $rowsGroupedByName = [];
         foreach ($rows as $row) {
```

Comment:
Here you can add an empty check on the `niet verplicht` column of the excel. After checking this you can assume that the field exists and has a value.